Error evaluating Problog model:
    cleanup, actions = node()  # Evaluate the node
                       ^^^^^^
  File "/Users/zhenzhili/miniforge3/envs/langda/lib/python3.11/site-packages/problog/eval_nodes.py", line 872, in __call__
    raise ArithmeticError(base_message, location)
problog.logic.ArithmeticError: Error while evaluating 'is'(4,6+4>6->2; -2+-1): Unknown function ';'/2 at 40:6.% Problog Inference Result：
goal_reached(1) = 0.1325
goal_reached(2) = 0.1688
goal_reached(3) = 0.1108
goal_reached(4) = 0.0950
goal_reached(5) = 0.0611
overrun_exception(1) = 0.0000
overrun_exception(2) = 0.0360
overrun_exception(3) = 0.0406
overrun_exception(4) = 0.0225
overrun_exception(5) = 0.0210% Problog Inference Result：
goal_reached(1) = 0.1325
goal_reached(2) = 0.1688
goal_reached(3) = 0.1108
goal_reached(4) = 0.0950
goal_reached(5) = 0.0611
overrun_exception(1) = 0.0000
overrun_exception(2) = 0.0360
overrun_exception(3) = 0.0406
overrun_exception(4) = 0.0225
overrun_exception(5) = 0.0210